README.md

```diff
@@ -50,6 +50,8 @@ environment.
    JupyterLab environment in development mode. Afterwards, `cd` back to the root
    of the repository.
 
+   - Please also run `jupyter labextension install @jupyter-widgets/jupyterlab-manager` if your extension uses ipywidget. Currently, big_query extension is using it.
```

Comment:
Can we accomplish this a better way? 

Can we just add a dependency `"@jupyter-widgets/jupyterlab-manager": "1.1",` to https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_bigquery/package.json

Reply:
I tried installing that from `npm install @jupyter-widgets/jupyterlab-manager@1.1`.  It did not work.  I think `jupyter labextension install` is doing some extra things to setup.  Did not find resources `jupyter labextension install` vs `npm install`.

Reply:
I can make this a sub-command in `jupyterlab_bigquery/package.json`.  However, I am personally against it for 2 reasons:
1) jupyter lab extension manager is kind not working so well.  Different jupyter versions needs different version of extension manager.  We can mess user's local extension manager if we install it here
2) other packages (ipywidgets) do not have the installed automatically.

Reply:
Ok I we will just leave it here, even though I wish there was a cleaner way.